subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/KubernetesReservationClient.kt

```diff
@@ -269,6 +273,11 @@ class KubernetesReservationClient(
             // https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application/#my-pod-stays-waiting
             if (!waitingMessage.isNullOrBlank()) {
                 logger.warn("Can't start pod on this node: $waitingMessage")
```

Comment:
What is "node" in the message? Is it k8s node?

Reply:
yes, but i don't understand why it's important here (probably same as you)

Reply:
Ok, let's rephrase to `Can't start pod: ...`